src/MaterialEditor.Base/PluginBase.cs

```diff
@@ -290,15 +291,17 @@ public class ShaderPropertyData
             public ShaderPropertyType Type;
             public string DefaultValue;
             public string DefaultValueAssetBundle;
+            public bool Hidden;
             public float? MinValue;
             public float? MaxValue;
 
-            public ShaderPropertyData(string name, ShaderPropertyType type, string defaultValue = null, string defaultValueAB = null, string minValue = null, string maxValue = null)
+            public ShaderPropertyData(string name, ShaderPropertyType type, string defaultValue = null, string defaultValueAB = null, string hidden = null, string minValue = null, string maxValue = null)
             {
                 Name = name;
                 Type = type;
                 DefaultValue = defaultValue.IsNullOrEmpty() ? null : defaultValue;
                 DefaultValueAssetBundle = defaultValueAB.IsNullOrEmpty() ? null : defaultValueAB;
+                Hidden = bool.TryParse(hidden, out bool result);
```

Comment:
This seems to be a mistake, the return value only shows if parsing was successful, not if the property is actually set to true.

Reply:
Maybe something like
```suggestion
                Hidden = bool.TryParse(hidden, out bool result) && result;
```

Reply:
Yeah, this is a mistake.